modules/mocked_authentication/spec/factories/mock_credential_infos.rb

```diff
@@ -5,4 +5,48 @@
     credential_info_code { SecureRandom.hex }
     credential_info { { credential: 'some-credential' } }
   end
+
+  factory :mock_data, class: Hash do
```

Comment:
We'll want to do a `fixture` for these rather than a `factory`, since they're really a rails class, but an 'external' entity